Installers/Linux/RUN/postinstall.sh

```diff
@@ -16,8 +16,18 @@ fi
 DIR=$(pwd)
 IDIR="/usr/lib/mono/xbuild/MonoGame/v3.0"
 
-# Automatic dependency installer
-./Dependencies/dependencies.sh
+# Show dependency list
+echo "Please make sure the following packages are installed:"
+echo " - monodevelop"
+echo " - libopenal-dev"
+echo " - referenceassemblies-pcl / mono-pcl"
+echo " - ttf-mscorefonts-installer / mscore-fonts"
+echo " - gtk-sharp3"
+read -p "Continue (Y, n): " choice2
```

Comment:
@dellis1972 Just about the question above, they get an option to cancel the installer if they don't have packages installed.